src/app/components/ContactView.js

```diff
@@ -44,17 +45,16 @@ const ContactView = ({ properties, contactID, errors }) => {
             <ContactSummary properties={properties} />
             <div className="pl1 pr1">
                 <ContactViewProperties contactID={contactID} properties={properties} field="email" />
-                {hasPaidMail ? (
-                    <>
-                        <ContactViewProperties contactID={contactID} properties={properties} field="tel" />
-                        <ContactViewProperties contactID={contactID} properties={properties} field="adr" />
-                        <ContactViewProperties contactID={contactID} properties={properties} />
-                    </>
-                ) : (
-                    <div className="mb1 pl1 pr1">
-                        <ContactPromote />
-                    </div>
-                )}
+                <div className="relative">
```

Comment:
Doing things this way, what if a free user edits the HTML code and simply erases the
```html
<div className="contactView-locked-background">
    <ContactPromote />
</div>
```
?
Will she have access to the locked properties? Or are these properties actually empty for a free user?

Reply:
That's OK.